common/components/SendButton.tsx

```diff
@@ -1,15 +1,15 @@
 import React from 'react';
 import { SendButtonFactory } from './SendButtonFactory';
 import translate from 'translations';
-import { ConfirmationModal } from 'components/ConfirmationModal';
+import { ConfirmationModalTemplate } from 'components/ConfirmationModalTemplate';
 
 export const SendButton: React.SFC<{
   onlyTransactionParameters?: boolean;
-  customModal?: typeof ConfirmationModal;
+  customModal?: typeof ConfirmationModalTemplate;
```

Comment:
This should be `typeof ConfirmationModal` where `ConfirmationModal` implements `ConfirmationModalTemplate`